source/_integrations/webmin.markdown

```diff
@@ -21,6 +21,14 @@ This {% term integration %} provides sensors for monitoring the CPU and memory u
 
 {% include integrations/config_flow.md %}
 
+<div class='note warning'>
+
+Please beware that the Webmin API ignores any 2FA set up via the frontend, e.g. it is even with enabled 2FA possible to use the API with only username and password.
+
+For best security, create a specific Webmin user for Home Assistant with a strong password and only "Can accept RPC calls" permission (under "Permissions for all modules") and remove the "Can accept RPC calls" permission for all other users.
```

Comment:
Isn't this just the best practice to begin with? And if so, should we make it a pre-requisite?

Reply:
Of course it is always better to have a dedicated user for that. What do you mean by making it a pre-requisite? Remove the paragraph because we assume that people are doing best practices and just inform that 2FA is ignored in API calls or put the information at another place?